fib.py

```diff
@@ -0,0 +1,21 @@
+def fibonacci(n):
+    """
+    This function returns the nth Fibonacci number.
+    The Fibonacci sequence is as follows: 0, 1, 1, 2, 3, 5, 8, 13, ...
+    So fibonacci(1) returns 0, fibonacci(2) returns 1, etc.
+    """
+    if n <= 0:
```

Comment:
Consider adding a check to ensure that 'n' is an integer. This will prevent unexpected behavior if a non-integer value is passed to the function. [important]